pkg/chains/storage/gcs/gcs.go

```diff
@@ -60,55 +62,32 @@ func NewStorageBackend(ctx context.Context, cfg config.Config) (*Backend, error)
 // StorePayload implements the storage.Backend interface.
 func (b *Backend) StorePayload(ctx context.Context, obj objects.TektonObject, rawPayload []byte, signature string, opts config.StorageOpts) error {
 	logger := logging.FromContext(ctx)
-	// TODO: Handle unsupported type gracefully
-	tr := obj.GetObject().(*v1beta1.TaskRun)
-	// We need multiple objects: the signature and the payload. We want to make these unique to the UID, but easy to find based on the
-	// name/namespace as well.
-	// $bucket/taskrun-$namespace-$name/$key.signature
-	// $bucket/taskrun-$namespace-$name/$key.payload
-	sigName := sigName(tr, opts)
```

Comment:
Should we remove `sigName` definition? It's only used by tests now it seems. Maybe move to the `_test.go` file? Same for `payloadName`.